java/libphonenumber/src/com/google/i18n/phonenumbers/Phonemetadata.java

```diff
@@ -0,0 +1,1119 @@
+/*
+ * Copyright (C) 2010 The Libphonenumber Authors
```

Comment:
I mean the Phonemetadata.java file that was there before we deleted it for nano.